recipes/batch_tar/meta.yaml

```diff
@@ -0,0 +1,75 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please remove all comments from the yams file.

Reply:
Thanks for the reminder!